docs/guides/10-integrating-react.md

```diff
@@ -0,0 +1,111 @@
+We are working on react bindings that offer a better user experience, meanwhile this guide contains the basic fundamentals of how to integrate Airhship components with [React.js](https://reactjs.org/).
```

Comment:
I'd present the two topics in the opposite way:

`This guide contains blah blah blah` and after that sentence `We are working on providing React bindings to offer a better developer experience in the future`